README.md

```diff
@@ -25,6 +25,27 @@ Based on traditional API Gateways, it intercepts all incoming messages moving th
 
 It supports all the protocols AsyncAPI supports through [bindings](https://github.com/asyncapi/bindings).
 
-This **Event Gateway** is also compatible with the HTTP protocol, natively or trough an external provider like [Krakend.io](http://krakend.io).
+This **Event Gateway** is also compatible with the HTTP protocol, natively or through an external provider like [Krakend.io](http://krakend.io).
 
-AsyncAPI also offers a Marketplace where users can install middlewares for the **Event Gateway** made by the community and also some officially maintained by the AsyncAPI org.
+AsyncAPI also offers a Marketplace where users can install middlewares for the **Event Gateway** made by the community or officially maintained by the AsyncAPI org.
```

Comment:
is this correct and valid atm? what marketplace? or you just mean a list of official and community middleware? like we have with generator templates, that there are official ones and not

Marketplace kinda implies monetization, no?

Reply:
> Marketplace kinda implies monetization, no?

Yeah, you are right. I believe Marketplace implies monetization. 

> or you just mean a list of official and community middleware?

Yes, I do. WDYT about using the word `Catalog` instead?

Reply:
`Catalog`? perfect 🚀

Reply:
I ended up removing the first mention to the Marketplace in favor of just the line we are talking about. Then simplified to `A catalog of middlewares made by the community is also available.`. It makes it simpler.